README.md

```diff
@@ -121,6 +121,14 @@ The value of the `refresh_token` field in the response is the offline token.
 
 All calls require an Authorization header.
 
+### Availability monitoring calls:
```

Comment:
You might want to refer to the spring official documentation here.

Reply:
good idea, done!